vipps-report-api-faq.md

```diff
@@ -49,6 +49,18 @@ The Report API has some benefits over this:
 * The data is in JSON format, making it easy to use in different ways.
   * The SFTP service offered XML, XSLX, CSV and PDF, but most users now want JSON.
 
+## Why does a merchant need to log in on portal.vipps.no to select accounting partner?
+
+An accounting partner gets access to the merchant's sales unit's data.
+We require the merchant to log in securely on
+[portal.vipps.no](https://portal.vipps.no)
+to provide this access and consent to sharing the sata with the accounting partner.
+
+## Can a partner specify the accounting partner for a sales unit?
+
+Nope. See
```

Comment:
```suggestion
No, please see
```

Reply:
I think "Nope" conveys a feeling of nonchalance and it gives me a bad vibe. 😬

Reply:
Ok, good., We have "Nope" a few other places, I think.